course/style.css

```diff
@@ -266,9 +277,6 @@ a {
 	text-decoration: none;
 }
 
-.wp-block-button {
-	text-align: center;
```

Comment:
Not really sure about this change. It shouldn't affect button's position, but the position of text inside a container.
And as we can see [here](https://github.com/Automattic/sensei/pull/6938#issuecomment-1570929148), buttons remain aligned by center.

Reply:
Okay, I see. Those buttons are anchors, and this change affects them.
But for some reason I don't see that change.
Maybe a cache issue :(

Reply:
Yes, it was cache :(